.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
:+1: Can someone take care of that? We can also remove the `.npmignore` that ignores the same thing. My recommendation is the one I already use globally:

```
# OS files
.DS_Store
.DS_Store?
.Spotlight-V100
.Trashes
Icon?
ehthumbs.db
Thumbs.db

# IDEs
.idea
.workspace
.metadata

# Node
node_modules
npm-debug.log

# Package managers
bower_components
```

Suggestions welcome.